<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>471</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Classfier</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>SVM Parameter</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gamma</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SVM_gamma_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>-1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SVM_c_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainClassifier_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Train Classfier</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Grid Search</string>
     </property>
     <widget class="QPushButton" name="gridTrainClassifier_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid Train</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gridTest_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid Test</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Model Test</string>
     </property>
     <widget class="QPushButton" name="loadSVM_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Load SVM .xml</string>
      </property>
     </widget>
     <widget class="QPushButton" name="accuracy_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Accuracy</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Machine Learning</string>
    </property>
    <widget class="QPushButton" name="machineLearning_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PCA and Combine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openFiles_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imageProcess_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Process</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PCA_checkBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PCA</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>22</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
